src/components/Game/Game.tsx

```diff
@@ -1,24 +1,59 @@
 'use client';
 
 import { useState } from 'react';
+import toast, { Toaster } from 'react-hot-toast';
 import { BiShuffle } from 'react-icons/bi';
 import { handleShuffle } from '@/utils/handleShuffle';
-import { GameT } from '@/models/Game';
 import { Button } from '../Button/Button';
 import { LetterGrid } from '../LetterGrid/LetterGrid';
 import styles from './game.module.scss';
 
+/* import { GameT } from '@/models/Game'; */
+
+type Word = {
+  word: string;
+};
+
+type Game = {
+  letters: string[];
+  levelScores: number[];
+  totalScore: number;
+  matchedWords: Word[];
+  panagrams: Word[];
+};
+
 type GameProps = {
-  game: GameT;
+  game: Game;
 };
 
 export const Game = ({ game }: GameProps) => {
-  const { letters } = game;
+  const { letters, matchedWords } = game;
   const [gameLetters, setGameLetters] = useState(letters);
   const [wordInput, setWordInput] = useState('');
+  const [foundWords, setFoundWords] = useState([]);
 
+  const matchesWithMainLetter = matchedWords.filter((word) =>
```

Comment:
We don't need this. If there are words that do not contain the the main letter this is a backend issue.